apps/journeys-admin/src/components/TemplateList/ActiveTemplateList/ActiveTemplateList.tsx

```diff
@@ -121,38 +125,51 @@ export function ActiveTemplateList({
     data?.journeys != null ? sortJourneys(data.journeys, sortOrder) : undefined
 
   return (
-    <>
+    <Box
+      sx={{
+        mt: { xs: 3, sm: 2 },
+        px: { xs: 5, sm: 0 }
+      }}
+    >
       {sortedJourneys != null ? (
         <Box>
```

Comment:
I dont think this box is actually needed, we can just turn it into a fragment. This way we reduce the amount of divs in the DOM https://react.dev/reference/react/Fragment
```suggestion
        <>
```